jordangollan.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<!--HAVE A NICE DAY AND MERRY CHRISTMAS 2021 :)-->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JWD7 Student</title>
+</head>
+<body>
+    <div class="card">
+        <img src="./images/jordangollanpic.jpg" alt="A photo of Jordan with text that says Thank You with a love heart" style="width: 100%" />
```

Comment:
Previously, the image was a photo of a brown lion, with the alt text description "Close-up photo of a Brown Lion from pexels.com."

The image file now contains a photo of myself with the alt text description "A photo of Jordan with text that says Thank You with a love heart."